presentation/src/main/java/com/london/presentation/screen/moiveDetalis/MovieDetailsViewModel.kt

```diff
@@ -0,0 +1,71 @@
+package com.london.presentation.screen.moiveDetalis
+
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.london.domain.usecase.GetMovieDetailsUseCase
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+import org.koin.android.annotation.KoinViewModel
+
+@KoinViewModel
+class MovieDetailsViewModel(
+private val getMovieDetailsUseCase: GetMovieDetailsUseCase
+) : ViewModel() , MovieDetailsIntersection {
```

Comment:
lets reformat the files before commit
```suggestion
) : ViewModel(), MovieDetailsIntersection {
```